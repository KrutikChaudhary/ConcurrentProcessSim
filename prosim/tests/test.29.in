8 3
Proc1 6 7
DOOP 5
LOOP 5
  BLOCK 1
  DOOP 4
END
HALT

Proc2 6 6
DOOP 5
LOOP 4
  BLOCK 2
  DOOP 5
END
HALT

Proc3 6 5
DOOP 5
LOOP 5
  BLOCK 1
  DOOP 4
END
HALT

Proc4 6 4
DOOP 5
LOOP 4
  BLOCK 2
  DOOP 5
END
HALT

Proc5 6 3
DOOP 2
LOOP 5
  BLOCK 5
  DOOP 1
END
HALT

Proc6 6 2
DOOP 2
LOOP 4
  BLOCK 6
  DOOP 2
END
HALT

Proc7 6 1
DOOP 2
LOOP 5
  BLOCK 5
  DOOP 1
END
HALT

Proc8 6 0
DOOP 2
LOOP 4
  BLOCK 6
  DOOP 2
END
HALT

