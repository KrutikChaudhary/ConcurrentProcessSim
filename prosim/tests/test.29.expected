00000: process 1 new
00000: process 1 ready
00000: process 2 new
00000: process 2 ready
00000: process 3 new
00000: process 3 ready
00000: process 4 new
00000: process 4 ready
00000: process 5 new
00000: process 5 ready
00000: process 6 new
00000: process 6 ready
00000: process 7 new
00000: process 7 ready
00000: process 8 new
00000: process 8 ready
00000: process 8 running
00002: process 8 blocked
00002: process 7 running
00004: process 7 blocked
00004: process 6 running
00006: process 6 blocked
00006: process 5 running
00008: process 8 ready
00008: process 5 blocked
00008: process 8 running
00009: process 7 ready
00010: process 8 blocked
00010: process 7 running
00011: process 7 blocked
00011: process 4 running
00012: process 6 ready
00012: process 4 ready
00012: process 6 running
00013: process 5 ready
00014: process 6 blocked
00014: process 5 running
00015: process 5 blocked
00015: process 4 running
00016: process 8 ready
00016: process 7 ready
00016: process 4 ready
00016: process 8 running
00018: process 8 blocked
00018: process 7 running
00019: process 7 blocked
00019: process 4 running
00020: process 6 ready
00020: process 5 ready
00020: process 4 ready
00020: process 6 running
00022: process 6 blocked
00022: process 5 running
00023: process 5 blocked
00023: process 4 running
00024: process 8 ready
00024: process 7 ready
00024: process 4 ready
00024: process 8 running
00026: process 8 blocked
00026: process 7 running
00027: process 7 blocked
00027: process 4 running
00028: process 6 ready
00028: process 5 ready
00028: process 4 blocked
00028: process 6 running
00030: process 4 ready
00030: process 6 blocked
00030: process 5 running
00031: process 5 blocked
00031: process 4 running
00032: process 8 ready
00032: process 7 ready
00032: process 4 ready
00032: process 8 running
00034: process 8 finished
00034: process 7 running
00035: process 7 blocked
00035: process 4 running
00036: process 6 ready
00036: process 5 ready
00036: process 4 ready
00036: process 6 running
00038: process 6 finished
00038: process 5 running
00039: process 5 blocked
00039: process 4 running
00040: process 7 ready
00040: process 4 ready
00040: process 7 running
00041: process 7 finished
00041: process 4 running
00043: process 4 blocked
00043: process 3 running
00044: process 5 ready
00044: process 3 ready
00044: process 5 running
00045: process 4 ready
00045: process 5 finished
00045: process 4 running
00048: process 4 ready
00048: process 4 running
00050: process 4 blocked
00050: process 3 running
00052: process 4 ready
00052: process 3 ready
00052: process 4 running
00055: process 4 ready
00055: process 4 running
00057: process 4 blocked
00057: process 3 running
00059: process 4 ready
00059: process 3 blocked
00059: process 4 running
00060: process 3 ready
00062: process 4 ready
00062: process 4 running
00064: process 4 finished
00064: process 3 running
00067: process 3 ready
00067: process 3 running
00068: process 3 blocked
00068: process 2 running
00069: process 3 ready
00069: process 2 ready
00069: process 3 running
00072: process 3 ready
00072: process 3 running
00073: process 3 blocked
00073: process 2 running
00074: process 3 ready
00074: process 2 ready
00074: process 3 running
00077: process 3 ready
00077: process 3 running
00078: process 3 blocked
00078: process 2 running
00079: process 3 ready
00079: process 2 ready
00079: process 3 running
00082: process 3 ready
00082: process 3 running
00083: process 3 blocked
00083: process 2 running
00084: process 3 ready
00084: process 2 ready
00084: process 3 running
00087: process 3 ready
00087: process 3 running
00088: process 3 finished
00088: process 2 running
00089: process 2 blocked
00089: process 1 running
00091: process 2 ready
00091: process 1 ready
00091: process 2 running
00094: process 2 ready
00094: process 2 running
00096: process 2 blocked
00096: process 1 running
00098: process 2 ready
00098: process 1 ready
00098: process 2 running
00101: process 2 ready
00101: process 2 running
00103: process 2 blocked
00103: process 1 running
00104: process 1 blocked
00105: process 2 ready
00105: process 1 ready
00105: process 2 running
00108: process 2 ready
00108: process 2 running
00110: process 2 blocked
00110: process 1 running
00112: process 2 ready
00112: process 1 ready
00112: process 2 running
00115: process 2 ready
00115: process 2 running
00117: process 2 finished
00117: process 1 running
00119: process 1 blocked
00120: process 1 ready
00120: process 1 running
00123: process 1 ready
00123: process 1 running
00124: process 1 blocked
00125: process 1 ready
00125: process 1 running
00128: process 1 ready
00128: process 1 running
00129: process 1 blocked
00130: process 1 ready
00130: process 1 running
00133: process 1 ready
00133: process 1 running
00134: process 1 blocked
00135: process 1 ready
00135: process 1 running
00138: process 1 ready
00138: process 1 running
00139: process 1 finished
Process 1: Run time 25, Block time 5, Wait time 109
Process 2: Run time 25, Block time 8, Wait time 84
Process 3: Run time 25, Block time 5, Wait time 58
Process 4: Run time 25, Block time 8, Wait time 31
Process 5: Run time 7, Block time 25, Wait time 13
Process 6: Run time 10, Block time 24, Wait time 4
Process 7: Run time 7, Block time 25, Wait time 9
Process 8: Run time 10, Block time 24, Wait time 0
